Python/compile.c

```diff
@@ -2646,48 +2657,24 @@ static int
 compiler_if(struct compiler *c, stmt_ty s)
 {
     basicblock *end, *next;
-    int constant;
     assert(s->kind == If_kind);
-    end = compiler_new_block(c);
+    next = end = compiler_new_block(c);
     if (end == NULL)
         return 0;
 
-    constant = expr_constant(s->v.If.test);
-    /* constant = 0: "if 0"
-     * constant = 1: "if 1", "if 2", ...
-     * constant = -1: rest */
-    if (constant == 0) {
-        BEGIN_DO_NOT_EMIT_BYTECODE
```

Comment:
You can remove this macro and the attribute in the compiler struct if is not used anymore